lib/three_scale/domain_substitution.rb

```diff
@@ -146,9 +127,7 @@ def external_domain
     #   root_url(host: account.external_admin_domain)
     #   # => "https://provider-admin.proxied-domain.com"
     def external_admin_domain
-      ThreeScale::Deprecation.silence do
-        ThreeScale::DomainSubstitution::Substitutor.to_external(admin_domain)
-      end
+      ThreeScale::DomainSubstitution::Substitutor.to_external(admin_domain)
```

Comment:
Don't we need to keep deprecation silence for both `admin_domain` and `domain`?

Reply:
What we decided is to maintain the deprecation instead of making admin_domain and domain private

Reply:
actually `#admin_domain` was a method so it can be made private + remove deprecation. The rest are just attributes and to avoid overriding , they remain deprecated. Or no?